attestation-agent/attester/README.md

```diff
@@ -0,0 +1,26 @@
+# Attester Plugins
+
+This crate provides attester plugins for different platforms. An [attester](https://www.rfc-editor.org/rfc/rfc9334.html#section-7.2) plugin can provide the following abilities
+
+- Get remote attestation evidence.
+- Extend runtime measurement.
+- Check the initdata binding.
+
+## Evidence Getter Tool
+
+This crate provides a simple tool to detect the current platform type and get related quote due to given report data.
+
+Build the binary
+```shell
+git clone https://github.com/confidential-containers/guest-components.git
+cd guest-components/attestation-agent/attester
+
+cargo build --features=bin --bin evidence_getter --release
```

Comment:
I had forgotten that I had a slightly patched `evidence_getter`. I believe this needs to be 
```suggestion
cargo build --features=bin,codicon --bin evidence_getter --release
```
maybe `Cargo.toml` needs updating too

Reply:
I have test the command in this PR locally and finished. `codicon` is a dependency required by `csv-attester` that is by default enabled by `default-features`

Reply:
Indeed, I had only `tdx-attester` enabled through local modifications...